tests/Test/AbstractFixerTestCase.php

```diff
@@ -193,7 +193,10 @@ final public function testFixerDefinitions(): void
         $dummyFileInfo = new StdinFileInfo();
 
         foreach ($samples as $sampleCounter => $sample) {
+            // @phpstan-ignore-next-line
             self::assertInstanceOf(CodeSampleInterface::class, $sample, \sprintf('[%s] Sample #%d', $fixerName, $sampleCounter));
+
+            // @phpstan-ignore-next-line
             self::assertIsInt($sampleCounter);
 
```

Comment:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/src/FixerDefinition/FixerDefinitionInterface.php#L34 ,
i wonder to simply:

```diff
-            // @phpstan-ignore-next-line
-            self::assertInstanceOf(CodeSampleInterface::class, $sample, \sprintf('[%s] Sample #%d', $fixerName, $sampleCounter));
-
-            // @phpstan-ignore-next-line
-            self::assertIsInt($sampleCounter);
-
```

Reply:
Done